components/style/properties/shorthand/font.mako.rs

```diff
@@ -4,17 +4,18 @@
 
 <%namespace name="helpers" file="/helpers.mako.rs" />
 
-<%helpers:shorthand name="font" sub_properties="font-style font-variant font-weight
+<%helpers:shorthand name="font" sub_properties="font-style font-variant font-weight font-stretch
                                                 font-size line-height font-family"
                     spec="https://drafts.csswg.org/css-fonts-3/#propdef-font">
-    use properties::longhands::{font_style, font_variant, font_weight, font_size,
+    use properties::longhands::{font_style, font_variant, font_weight, font_stretch, font_size,
                                 line_height, font_family};
```

Comment:
Generally in servo, every use declaration have to be in one line. If line is too long, we should move some of it to another use declaration. Like this:
```rust
use properties::longhands::{font_style, font_variant, font_weight, font_stretch};
use properties::longhands::{font_size, line_height, font_family};
```
Probably it's overlooked during review. Since you're editing these lines, it can be nice if you fix this too :) Not a big deal though